pkg/occlient/occlient.go

```diff
@@ -843,12 +815,66 @@ func (c *Client) BootstrapSupervisoredS2I(name string, builderImage string, labe
 	return nil
 }
 
+// AddBootstrapInitContainer adds the bootstrap init container to the deployment config
+// dc is the deployment config to be updated
+// dcName is the name of the deployment config
+func (c *Client) AddBootstrapInitContainer(dc *appsv1.DeploymentConfig, dcName string) {
+	appRootVolumeName := fmt.Sprintf("%s-s2idata", dcName)
```

Comment:
appRootVolumeName is constructed in the multiple places. Could we generate this name in one place?

Reply:
OK I will create a private function to fetch this